src/Teams/Data/ProgramTestRepos/IProgramTestRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Teams.Domain;
+
+namespace Teams.Data.ProgramTestRepos
+{
+    public interface IProgramTestRepository
+    {
+        List<ProgramTest> GetProgramCodeTests(Guid id);
```

Comment:
I would suggest using interfaces instead of implementations in method signatures everywhere where it's possible.
It makes our code more flexible. Please, use IList<ProgramTest> or ICollection<ProgramTest> here.